expression/builtin_miscellaneous.go

```diff
@@ -318,12 +318,12 @@ func (b *builtinIsIPv4Sig) eval(row []types.Datum) (d types.Datum, err error) {
 }
 
 // mysql 'is_ipv4' don't accept:
-//   1. mapped IPv6 address such as '::ffff:1.2.3.4', while 'IP.To4' in pcakge 'net' do
-//   2. number-skipped IPv4 address such as '192..1.1', while system call 'inet_aton' do
+//   1. mapped IPv6 address such as '::ffff:1.2.3.4', while 'IP.To4' in pcakge 'net' do.
```

Comment:
hi, @spongedu 
1. INET_ATON() is also a builtin func in MySQL, and as the [manual](https://dev.mysql.com/doc/refman/5.7/en/miscellaneous-functions.html#function_is-ipv4) says, IS_IPV4() checks more strict than INET_ATON(), `comment 2` is unnecessary for this function, we only need to know what format this function would check. 

2. for `comment 1`, I think it can be rewrite as
`isIPv4 checks IPv4 address which satisfying the format A.B.C.D(0<=A/B/C/D<=255), 
mapped IPv6 address like '::ffff:1.2.3.4' would return false.`

3. isIPv4 checks whether an IP address is an IPv4 address which follows a strict format, it shouldn't be affected by where it is or which function it is invoked by. So, remove line 320, and rename mysqlIsIPv4 to `isIPv4` may be better.